azurerm/internal/services/monitor/resource_arm_monitor_metric_alert.go

```diff
@@ -166,7 +166,7 @@ func resourceArmMonitorMetricAlert() *schema.Resource {
 			"auto_mitigate": {
 				Type:     schema.TypeBool,
 				Optional: true,
-				Default:  false,
+				Default:  true,
```

Comment:
@sivanguetta thanks for this PR - just checking - is this behaviour mirrored in the Azure CLI/Portal? Thanks!

Reply:
> @sivanguetta thanks for this PR - just checking - is this behaviour mirrored in the Azure CLI/Portal? Thanks!

In Azure portal - 'auto_mitigate' is always true, the user cannot change this property.

In Azure CLI - in case that 'auto_mitigate' property was not provided, the metric alert rule is created without 'auto_mitigate' property (it is an optional property). 
When metric alert rule is created without 'auto_mitigate', it is considered as 'true'. 

As there was no indication what is the behavior when auto_mitigate property is not provided, I have raised a PR to update the documentation- https://github.com/Azure/azure-rest-api-specs/pull/8412

Reply:
@sivanguetta makes sense to me 👍

Reply:
> @sivanguetta makes sense to me 👍

@tombuildsstuff 
Thank you for the review:-)
I see that all checks have passed but I can't merge the pull request because I'm not authorized.
How can I merge the PR? what is missing?